src/main/sql/block_detail.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE block_detail                                                          
+(                                                                              
+--Primary Key                                                                                         
+  PRIMARY KEY (block_id,entry_number),                                             
+                                                                         
+--Columns                                                              
+  block_id INT NOT NULL,
+  entry_number INT NOT NULL,
+  cash_value NUMERIC(10,2) NOT NULL DEFAULT 0,
+  user_id (whatever format user_detail has),
+  block_detail_description VARCHAR(200) NOT NULL DEFAULT ' ',
+  written_off CHAR(1) NOT NULL DEFAULT 'Y',
+  currency_code CHAR(3) NOT NULL,
+  ledger_id CHAR(10) NOT NULL,
+
+--Constraints
+  CONSTRAINT written_off_is_boolean
```

Comment:
Maybe call this constraint something else? Technically the field isn't a boolean type